usecase/usecase-apptools-tests/tests.full.xml

```diff
@@ -83,6 +83,98 @@
           <test_script_entry>/opt/usecase-apptools-tests/samples/AppTools_arm/index.html</test_script_entry>
         </description>
       </testcase>
+      <testcase component="Crosswalk Use Cases/AppTools" execution_type="manual" id="AppTools_install_crosswalk_app_x86" platform="android" priority="P0" purpose="AppTools_install_crosswalk_app_x86 Test" status="designed" type="compliance">
+        <description>
+          <pre_condition>Please refer to README file, and remove environment PATH with 'crosswalk-app-tools/src/crosswalk-app'</pre_condition>
```

Comment:
Why refer to README file? To do what? I think crosswalk-app installation should be a precondition. And remind user to handle the confliction between local env PATH and installed crosswalk-app.